### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by krolich500grn**
[DEMO LINK](https://krolich500grn.github.io/react_article/)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -8,8 +9,10 @@ const article = {
   date: 'August 30, 2019',
 };
 
+const { title, text, date } = article;
+
 const App = () => (
-  <h1>React Article</h1>
+  <Article title={title} text={text} date={date} />
```
Comment (jstmpelowycz) [inline src/App.js:15]:
```suggestion
  <Article {...article} />
```

File: src/components/Article.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => (
```
Comment (jstmpelowycz) [inline src/components/Article.jsx:8]:
```suggestion
export const Article = ({
  title, date, text,
}) => (
```

**Review by jstmpelowycz — APPROVED** [review]
😃

File: src/App.js (hunk 1 above)
Comment (krolich500grn) [reply src/App.js:15]: Good point, thanks!

File: src/App.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article {...article} />
```
Comment (Ihor-Karpyn) [inline src/App.js:12]:
```suggestion
  <Article article={article} />
```

File: src/components/Article.jsx (hunk 2 above)
Comment (Ihor-Karpyn) [inline src/components/Article.jsx:8]:
```suggestion
export const Article = ({ title, date, text }) => (
```

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**